src/firefly/java/edu/caltech/ipac/table/io/VoTableReader.java

```diff
@@ -163,6 +164,12 @@ private static DataGroup[] voToDataGroups(VOElement docRoot,
         return handler.getAllTable();
     }
 
+    public static boolean isVoTable(File inFile) {
+        try {
+            DataGroup[] tbl = voToDataGroups(inFile.getAbsolutePath(), true, 0);
+            return tbl != null && tbl.length > 0;
+        } catch (Exception e) { return false; }
+    }
```

Comment:
This function appears to never be called. My bigger question is that is seems to go through code the retrieves the file. That is unexpected.

Reply:
It’s intended to read only the header, similar to IpacTableReader. However, after reviewing our guesser, I’m confident it does a good enough job detecting VOTable. As a result, this is no longer needed, and I’ll remove it.